src/App.js

```diff
@@ -1,23 +1,70 @@
-import logo from './logo.svg';
 import './App.css';
+import { useState } from 'react';
+import foodsJSON from '../src/foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
 
 function App() {
+
+  //const foodArr = [...foodsJSON];
+
+  const [foodList, setFoodList] = useState(foodsJSON)
+
+  const addFood = (recipe) =>{
+    const foodListCopy = [...foodList, recipe]
+    setFoodList(foodListCopy)
+  }
+  
+  const searchRecipes = (query) => {
+    const filteredRecipes = foodsJSON.filter((recipe) =>
+      recipe.name.toLowerCase().includes(query.toLowerCase())
+    );
+    setFoodList(filteredRecipes);
+  };
+
+  const deleteRecipe = (recipe) => {
```

Comment:
Here, in the `deleteRecipe` function, you are using the `foodsJSON` array to filter out the recipe to delete but this is not updated when a recipe is added or deleted. Instead, you should use the `foodList` state variable that you are updating when a recipe is added or filtered. So you need to replace `foodsJSON` with `foodList` in the `deleteRecipe` function.

```
const deleteRecipe = (recipe) => {
  const filteredRecipes = foodList.filter((food) => {
    return food.name !== recipe;
  });
  setFoodList(filteredRecipes);
};
```